src/components/CompareResults/TableHeader.tsx

```diff
@@ -72,19 +77,29 @@ function FilterableColumn({
             theme.palette.mode == 'light'
               ? Colors.Background200
               : Colors.Background200Dark,
-          borderRadius: '4px',
-          fontSize: 'inherit',
+          borderRadius: 0.5,
+          padding: '6px 12px',
         })}
       >
-        {name}
         <FilterListIcon
           fontSize='small'
           color={hasFilteredValues ? 'primary' : 'inherit'}
-          sx={{ marginInlineStart: 1 }}
+          sx={{ marginInlineStart: 1, marginRight: 0.5, marginLeft: 0 }}
           titleAccess={
             hasFilteredValues ? 'Some filters are active' : 'No active filters'
           }
         />
```

Comment:
@julienw There seems to be some misunderstanding here. 

Dasha's design does include _the arrow down in the table header._ 

<img width="928" alt="figma-design" src="https://github.com/user-attachments/assets/623031c9-2a62-4688-aa52-d64928e5205a">

My deployed url link design:
![image](https://github.com/user-attachments/assets/a4f611f7-284d-42cc-89a2-aef349da608e)

Reply:
> Dasha's design does include the filter icon in the table header.

Not her latest design for this :-) see slack message from June 17 and the linked design: https://www.figma.com/design/LRV76asSVOULlsWbRNZYp0/PerfCompare?node-id=3431-43375&node-type=section&m=dev

Yes I agree it's confusing because all the other designs are including the icon...